Black-Market/Black-Market/Scenes/SignIn/SignInView.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  SignInView.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 7/12/23.
+//
+
+import SwiftUI
+
+struct SignInView: View {
+    enum Dest: Hashable {
+        case home, signUp
+    }
+    @State private var password = ""
+    @State private var email = ""
+    @State private var path: [Dest] = []
+    @State private var isEmailValid: Bool = false
+    @State private var isPasswordValid: Bool = false
+    @State private var style: MarketButton.Style = .disabled
```

Comment:
Did you consider using a view model or any other patter to keep the bizz logic separated?

Reply:
Done